plugins/grpc/node/v1.12.0/buf.plugin.yaml

```
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/grpc/node
+plugin_version: v1.12.0
+source_url: https://github.com/grpc/grpc-node
+description: Generates Node client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/js:v3.21.2
+output_languages:
+  - javascript
+registry:
+  npm:
+    import_style: commonjs
+    deps:
+    - package: '@grpc/grpc-js'
+      version: '^1.7.0'
```

```suggestion
      version: '^1.7.3'
```

This is the version of grpc-js in grpc-node 1.12.0 release: https://github.com/grpc/grpc-node/compare/grpc-tools%401.11.3...grpc-tools%401.12.0#diff-30ec6aa4695ac502042ff6338a8d72b2e1d9a149f5e2d5e35b702146a9c6f2c1